main.tf

```diff
@@ -395,6 +395,7 @@ resource "aws_instance" "builder" {
 
     connection {
       host = coalesce(self.public_ip, self.private_ip)
+      user = local.build_info[each.key].platform.connection_user_builder
```

Comment:
Interesting... It's not being inherited from the `connection` block on the resource, line 358? That's how it is _supposed_ to work...

https://github.com/plus3it/terrafirm/pull/155/files#diff-dc46acf24afd63ef8c556b77c126ccc6e578bc87e3aa09a931f33d9bf2532fbbR358

Reply:
well i don't see the harm in the change. weird that the other options aren't needed though.